packages/react-components/react-nav-preview/src/components/NavDrawer/renderNavDrawer.tsx

```diff
@@ -5,15 +5,14 @@ import { assertSlots } from '@fluentui/react-utilities';
 import type { NavDrawerState } from './NavDrawer.types';
 import { NavContextValues } from '../NavContext.types';
 import { NavProvider } from '../NavContext';
-import { InlineDrawerSlots } from '@fluentui/react-drawer';
+import { DrawerSlots } from '@fluentui/react-drawer';
 
 /**
  * Render the final JSX of NavDrawer
  */
 export const renderNavDrawer_unstable = (state: NavDrawerState, contextValues: NavContextValues) => {
-  assertSlots<InlineDrawerSlots>(state);
+  assertSlots<DrawerSlots>(state);
```

Comment:
If you go with my other comment
```suggestion
```

Reply:
And you would also remove the jsxRuntime comments on the top

Reply:
🫡

Reply:
I ended up having to add the comments back to pass the CI builds.